guides/04_Quality_Bandwidth_Connectivity.md

```diff
@@ -55,6 +55,23 @@ The Amazon Chime SDK for JavaScript produces several kinds of events on the [Aud
 |------------ | ------------- | ------------- |
 |[metricsDidReceive](https://github.com/aws/amazon-chime-sdk-js/blob/bf6d01e236445684601e24f3e319dede728b5113/src/audiovideoobserver/AudioVideoObserver.ts#L76) | Exposes the WebRTC getStats metrics. There may be differences among browsers as to which metrics are reported. | All |
 
+> Note: We have built a video WebRTC statistics widget in the demo browser meeting app. This widget is shown as an overlay on the video, when the video is enabled.
+>
+> You get the following WebRTC stats in the video stats widget:
+>
+> Upstream video information: Resolution, Bitrate (kbps), Packets Sent, Frame Rate (fps).
+>
+> Downstream video information: Resolution, Bitrate (kbps), Packet Loss (%), Frame Rate (fps).
+>
+> You can check the implementation in the [demo app](https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/app/meetingV2/meetingV2.ts) to build your own custom widget (look for `getAndShowWebRTCStats` method in the demo). This video stats widget is built using the [getRTCPeerConnectionStats](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideocontroller.html#getrtcpeerconnectionstats) and the [metricsDidReceive](https://github.com/aws/amazon-chime-sdk-js/blob/bf6d01e236445684601e24f3e319dede728b5113/src/audiovideoobserver/AudioVideoObserver.ts#L76) APIs. Currently, there is no direct API in the amazon-chime-sdk-js for a local user to know if the simulcast resolution changes on the remote side. But, based on these stats you are seeing for any particular video tile (remote or local), you can check its resolution, bitrate and then decide if want to put an avatar on that video tile and remove when the resolution gets good or up to your expectation.
```

Comment:
I would remove/reword the text starting with "Currently, there is no direct..."
We are planning on adding an API to provide that information, it just isn't available quite yet.  Also I think we should be more generic about what they may want to do.  Maybe something like:

Through the information provided in these stats, the application can also monitor key attributes and take action.  For instance if the bitrate or resolution falls below a certain threshold, the user could be notified in some manner, or diagnostic reporting could take place.

Reply:
Ok, I have updated the working as suggested thanks!